src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogStorage.java

```diff
@@ -48,22 +50,35 @@ public OtelLogStorage(@Nonnull BuildInfo buildInfo, @Nonnull Tracer tracer, @Nul
 
     @Nonnull
     @Override
-    public BuildListener overallListener() {
+    public BuildListener overallListener() throws IOException, InterruptedException {
```

Comment:
I am concerned about these new Exceptions. They are throwing up without any explanation of what happened

Reply:
Resolving as @kuisathaverat said in a comment I can no longer find that this comment is outdated.